pymilo/transporters/lossfunction_transporter.py

```diff
@@ -42,15 +42,15 @@ def serialize(self, data, key, model_type):
     def deserialize(self, data, key, model_type):
         """
         Deserialize the special loss_function_ of the SGDClassifier, SGDOneClassSVM, Perceptron and PassiveAggressiveClassifier.
-        
-        the associated loss_function_ field of the pymilo serialized model, is extracted through the SGDClassifier's _get_loss_function function 
+
+        the associated loss_function_ field of the pymilo serialized model, is extracted through the SGDClassifier's _get_loss_function function
         with enough feeding of the needed inputs.
```

Comment:
Merge them into one line or justify them by putting pieces of first line into second line.